.github/workflows/regression.yml

```diff
@@ -81,3 +81,68 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           message: ${{ steps.report.outputs.result }}
+
+  regression-postgres:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Deps
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.19.2'
+      - name: Dir Setup
+        run: mkdir -p ~/go/bin
+
+      # Master branch
+      - name: Checkout master branch
+        uses: actions/checkout@v2
+        with:
+          ref: 'master'
+      - name: Check cache for master
+        uses: actions/cache@v2
+        id: cache_master
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Build Master
+        run: make build . && cp drand ~/go/bin/drand-existing
+
+      # Candidate branch
+      - name: Checkout candidate branch
+        uses: actions/checkout@v2
+      - name: Check cache for candidate
+        uses: actions/cache@v2
+        id: cache_candidate
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Build Candidate
+        run: make build && cp drand ~/go/bin/drand-candidate
+
+      # Regression test
+      - name: Run regression with PostgresDB backend
+        id: regression-bolt
```

Comment:
i think this id should be `regression_postgres` or similar

Reply:
Good catch!